src/libs/actions/IOU.js

```diff
@@ -993,12 +1001,22 @@ function getUpdateMoneyRequestParams(transactionID, transactionThreadReportID, t
         transactionID,
     };
 
+    const hasPendingWaypoints = _.has(transactionChanges, 'waypoints');
+    if (hasPendingWaypoints) {
+        updatedTransaction = {
+            ...updatedTransaction,
+            amount: CONST.IOU.DEFAULT_AMOUNT,
+            modifiedAmount: CONST.IOU.DEFAULT_AMOUNT,
+            modifiedMerchant: Localize.translateLocal('iou.routePending'),
+        };
+    }
```

Comment:
NAB: As I said elsewhere, I'm not sure we want this to go blank when editing. I think it's enough to have it grayed out and pending.

Reply:
Answered here: https://github.com/Expensify/App/pull/34135#issuecomment-1924805419

Reply:
Ok sounds good, thanks.